tests/Testcontainers.WebDriver.Tests/WebDriverContainerTest.cs

```diff
@@ -0,0 +1,31 @@
+namespace Testcontainers.WebDriver;
+
+using System;
+using Xunit.Abstractions;
+
+public class WebDriverContainerTest : IAsyncLifetime
+{
+  private readonly ITestOutputHelper testOutputHelper;
```

Comment:
I woud favor the container logger (`_webDriverContainer.Logger`), but only writes to the debug output (not test output).

Reply:
O.k